source/components/the-component-life-cycle.md

```diff
@@ -0,0 +1,48 @@
+When a component is invoked, a number of life-cycle hooks are triggered, in sequence, to properly initialize and render the component object and its corresponding DOM element.  The following hooks are a few of the most useful, for current apps.
+
+## init()
```

Comment:
Already documented at http://guides.emberjs.com/v2.1.0/object-model/classes-and-instances/#toc_initializing-instances